src/backend/SentenceGenerator.ts

```diff
@@ -361,7 +377,20 @@ export class SentenceGenerator {
                                         // its name for the output.
                                         const tokenName = this.lexerData.vocabulary.getSymbolicName(token);
                                         if (tokenName) {
-                                            const mapping = this.ruleMappings?.[tokenName];
+                                            const mappingValue = this.ruleMappings?.[tokenName];
+                                            let mapping: string | undefined;
+
+                                            if (Array.isArray(mappingValue)) {
+                                                if (mappingValue.length < 0) {
```

Comment:
same here

Reply:
Not readme, but here: https://github.com/mike-lischke/vscode-antlr4/blob/master/doc/sentence-generation.md

Reply:
Documentation with explanation and example